azure-samples/src/test/java/com/microsoft/azure/management/samples/ComputeSampleTests.java

```diff
@@ -86,6 +88,12 @@ public void testManageVirtualMachine() {
     }
 
     @Test
+    @Ignore("Recording fails due to a failure on Managed disks backend")
```

Comment:
it may be just a question of resolving merge conflicts - i believe I already fixed all the ComputeSampletests